jctools-experimental/src/main/java/org/jctools/channels/proxy/ProxyChannelFactory.java

```diff
@@ -271,7 +278,12 @@ private static void implementProcess(ClassVisitor classVisitor,
 
         // switch(UnsafeAccess.UNSAFE.getInt(rOffset)) // start with case 1, increment by 1; represents "type"
         getUnsafe(methodVisitor, int.class, localIndexOfROffset, 0);
-        methodVisitor.visitTableSwitchInsn(1, cases.length, endOfSwitch, cases);
+        methodVisitor.visitVarInsn(Opcodes.ISTORE, localIndexOfTypeId);
+        methodVisitor.visitVarInsn(Opcodes.ILOAD, localIndexOfTypeId);
```

Comment:
Assign the result of UnsafeAccess.UNSAFE.getInt(rOffset) to a local variable to aid debugging of the generated proxy